marker_api/src/ast/expr.rs

```diff
@@ -377,7 +377,7 @@ mod test {
         assert_eq!(24, size_of::<CharLitExpr<'_>>(), "CharLitExpr<'_>");
         assert_eq!(24, size_of::<BoolLitExpr<'_>>(), "BoolLitExpr<'_>");
         assert_eq!(88, size_of::<BlockExpr<'_>>(), "BlockExpr<'_>");
-        assert_eq!(88, size_of::<ClosureExpr<'_>>(), "ClosureExpr<'_>");
+        assert_eq!(72, size_of::<ClosureExpr<'_>>(), "ClosureExpr<'_>");
```

Comment:
I recommend you to use a snapshot-testing library.
I've been using [expect-test](https://docs.rs/expect-test/latest/expect_test/), which is a suprisingly low-overhead crate for snapshot testing.

This way you can capture all the data bout the types and their sizes in a string snapshot (inline our out-of-line in a separate file) and re-generate it if something changes with `UPDATE_EXPECT=1 cargo test`, which means maintaining tests becomes almost a zero-effort task, because they rewrite themselves, and you just review the diff.

This testing library was originally developed for use in `rust-analyzer` by [`@matklad`](https://github.com/matklad) and I've been using it in production successfuly so far. The one thing is that the library is barebones, and requires some small amount of batteries (like I added [in my repo here](https://github.com/Veetaha/snowpity/tree/master/crates/test-bat/src)) to make JSON snapshots and auto-format snapshots such that they don't exceed 80 lines in the editor

Reply:
Thank you for the suggestion. I created #202 to investigate this at one point :)